streams/src/test/java/org/apache/kafka/streams/StreamsConfigTest.java

```diff
@@ -518,8 +530,18 @@ public void shouldThrowIfBuiltInMetricsVersionInvalid() {
     }
 
     @Test
-    public void shouldResetToDefaultIfConsumerIsolationLevelIsOverriddenIfEosEnabled() {
+    public void shouldResetToDefaultIfConsumerIsolationLevelIsOverriddenIfEosAlphaEnabled() {
```

Comment:
We "duplicate" test for eos-alpha and eos-beta (similar pattern below for other tests)